JSDemos/Demos/List/ItemTemplate/description.md

```diff
@@ -1 +1,5 @@
-This demo shows how you can apply a&nbsp;custom template to&nbsp;items in&nbsp;the **List** UI component.
\ No newline at end of file
+To customize list items, you can use a default item template or implement a custom template.
+
+A default template is a set of data source fields that the List automatically recognizes. These fields are listed in the [items](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) section. To use the default template for customization, simply add or remove the fields to or from data objects. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) article for more information.
```

Comment:
- "simply" and other adverbs should be avoided
- "add or remove the fields to or from" You do not need to add 'to or from' - **from** is sufficient here since it goes with **remove**

```suggestion
A default template consists of a set of data source fields that the List automatically recognizes. These fields are listed in the [items](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) section. To use a default template for customization, add or remove the fields from data objects. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) article for more information.
```